ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -7,11 +7,22 @@
 "Accessibility.ActivityIndicator.Animating.label" = "In progress";
 "Accessibility.ActivityIndicator.Stopped.label" = "Progress halted";
 
+/* Accessibility alert for common use */
 "Accessibility.Alert" = "Alert";
+/* Accessibility dismiss label for common use */
 "Accessibility.Dismiss.Label" = "Dismiss";
+/* Accessibility dismiss hint for common use */
 "Accessibility.Dismiss.Hint" = "Double tap to dismiss";
+/* Accessibility done label for common use */
 "Accessibility.Done.Label" = "Done";
+/* Accessibility multi select hint for common use */
 "Accessibility.MultiSelect.Hint" = "Double tap to toggle selection";
+/* Accessibility select hint for common use */
```

Comment:
you are adding strings that didn't existed before?

Reply:
Fixed, it's a conflict issue.